container_build.sh

```diff
@@ -41,10 +41,13 @@ add_entrypoint() {
     containerfile=$(mktemp)
     cat > "${containerfile}" <<EOF
 FROM $2
-ENTRYPOINT [ "/usr/bin/$3.sh" ]
+ENTRYPOINT [ "/usr/bin/$3" ]
 EOF
     echo "$1 build --no-cache -t $2-$3 -f ${containerfile} ."
     eval "$1 build --no-cache -t $2-$3 -f ${containerfile} ."
+    # verify entrypoints created correctly by running help on them.
```

Comment:
I hope this manipulating of Containerfiles will work on konflux.

Reply:
We will have to break it out into something Konflux will understand or just add additional Containerfiles.

The issue is we have the same pattern for each different GPU type.